app/mailers/noisy_workflow.rb

```diff
@@ -23,11 +23,12 @@ def skip_review(action, recipient_email)
   def request_fact_check(action, recipient_email)
     @edition = action.edition
     fact_check_address = @edition.fact_check_email_address
+    fact_check_prefix = ENV.fetch("FACT_CHECK_SUBJECT_PREFIX", "")
```

Comment:
Is there some way we can get this from `Publisher::Application.fact_check_config` directly? It feels like something that might be forgotten if we decide to change this configuration somehow.